.azurepipelines/BaseTools-Build-For-Publication.yml

```diff
@@ -1,11 +1,19 @@
-# File BaseTools-Build-For-Publication.yml
-#
+## @file
 # Azure Pipeline to construct the final Nuget package for a binary release
 # of BaseTools.
 #
 # Copyright (C) Microsoft Corporation
 # SPDX-License-Identifier: BSD-2-Clause-Patent
 ##
+
+resources:
+  repositories:
+    - repository: mu_devops
+      type: github
+      endpoint: makubacki
+      name: makubacki/mu_devops
```

Comment:
friendly reminder to set this make to the official before merging

Reply:
Thanks, I updated that.